ext_localconf.php

```diff
@@ -3,7 +3,9 @@
 defined('TYPO3_MODE') or die();
 
 (function () {
-    $lockFactory = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(\TYPO3\CMS\Core\Locking\LockFactory::class);
-    $lockFactory->addLockingStrategy(\B13\DistributedLocks\RedisLockingStrategy::class);
+    if (!empty($GLOBALS['TYPO3_CONF_VARS']['SYS']['locking']['redis']['disabled'])) {
```

Comment:
@baschny, shouldn't that be just `empty()` without the negation?

Reply:
oh my, you are right. I had that called "enabled" at first, I didnt noticed. Good catch!